File: codes_python/A1-023.py
t = int(input())
unit = input().strip().upper()
if unit == "C":
    if t <= 0:
        state = "solid"
    elif t >= 100:
        state = "gas"
    else:
        state = "liquid"
else:
    if t <= 32:
        state = "solid"
    elif t >= 212:
        state = "gas"
    else:
        state = "liquid"
print(state)

File: codes_c/A1-023.c
#include <stdio.h>
int main()
{
    int t;
    char u;
    scanf("%d %c", &t, &u);
    if (u == 'C' || u == 'c')
    {
        if (t <= 0)
            printf("solid\n");
        else if (t >= 100)
            printf("gas\n");
        else
            printf("liquid\n");
    }
    else
    {
        if (t <= 32)
            printf("solid\n");
        else if (t >= 212)
            printf("gas\n");
        else
            printf("liquid\n");
    }
    return 0;
}
